companies/models.py

```diff
@@ -9,6 +9,13 @@ class Company(TimeStampModel):
     class Meta:
         db_table = 'companies'
 
+class CompanyImage(TimeStampModel):
+    company_image = models.URLField(max_length=500)
```

Comment:
이미 CompanyImage class의 member 변수이므로, `Image_url` 혹은 `url`만으로 충분할 것 같습니다.
```suggestion
    image_url = models.URLField(max_length=500)
```